src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/ExpressionCompilerTests.cs

```diff
@@ -2335,9 +2335,9 @@ static void Main()
         }
 
         /// <summary>
-        /// Flow analysis should be run on the generated method.
+        /// Flow analysis is not run on the generated method.
```

Comment:
I was of the impression that this is a deliberate optimization for the EE.  Is that not the case?